pootle/static/css/editor.css

```diff
@@ -1022,13 +1028,43 @@ html[dir="rtl"] .suggestion-original
 .editor-area-wrapper
 {
     background-color: #fff;
-    border: 1px solid #d9d9d9;
-    border-radius: 3px;
     box-shadow: 0 1px 3px 0 rgba(0, 0, 0, 0.075) inset;
-    margin: 0 auto 0.5em;
```

Comment:
We need the bottom margin for a bit of fresh air, it won't look good otherwise:
![screen shot 2016-09-08 at 11 05 36](https://cloud.githubusercontent.com/assets/16768/18343576/38a6a294-75b4-11e6-846f-31411f9ba065.png)